### ISSUE TITLE
[Wishlist] ability to define custom WPM as parameter

### ISSUE THREAD

**Issue by Yenthe666** [alanhamlett/readtime#4]
Hi @alanhamlett,

First of all: a big thank you for the package!
I think it would be a nice addition to make the WPM parameter configurable. Something along the lines of:
```
result = readtime.of_text('The shortest blog post in the world!', wpm=200)
```

This would allow you to calculate and get read times as you please. Medium uses 265 WPM but what about technical tutorials or advanced subjects? The WPM will go down and right now there is no easy way to change/define it.

**Comment by alanhamlett** [comment]
Implemented with 58821172e7b686a55416e3033b7403cb33244225 and released in v1.1.0.

**Comment by Yenthe666** [comment]
Awesome, this is a great extra addition! Thanks a lot @alanhamlett, really appreciate it 🙏
